src/js/common/schemaform/review/ReviewCollapsibleChapter.jsx

```diff
@@ -97,17 +70,44 @@ export default class ReviewCollapsibleChapter extends React.Component {
         <div id={`collapsible-${this.id}`} className="usa-accordion-content">
           {this.props.pages.map(page => {
             const editing = this.props.data[page.pageKey].editMode;
+            // Our pattern is to separate out array fields (growable tables) from
+            // the normal page and display them separately. The review version of
+            // ObjectField will hide them in the main section.
+            const arrayFields = getArrayFields(page);
+            // If a page is just an array, then we should skip rendering it, since
+            // we'll show the array separately
+            const hasNonArrayFields = hasFieldsOtherThanArray(page.schema);
 
             return (
               <div key={page.pageKey} className="form-review-panel-page">
                 <Element name={`${page.pageKey}ScrollElement`}/>
-                <FormPage
-                    reviewPage
-                    hideTitle={this.props.pages.length === 1}
-                    onEdit={() => this.handleEdit(page.pageKey, !editing)}
-                    onSubmit={() => this.handleSave(page.pageKey)}
-                    reviewMode={!editing}
-                    route={{ pageConfig: page }}/>
+                {hasNonArrayFields &&
+                  <FormPage
+                      reviewPage
+                      hideTitle={this.props.pages.length === 1}
+                      onEdit={() => this.handleEdit(page.pageKey, !editing)}
+                      onSubmit={() => this.handleEdit(page.pageKey, false)}
+                      reviewMode={!editing}
+                      route={{ pageConfig: page }}>
+                    {!editing ? <div/> : <ProgressButton
```

Comment:
The library displays its own submit button if you don't pass any children to the `Form` component. So this works around that.